media/js/mozorg/home/home-b.js

```diff
@@ -77,20 +86,27 @@
                     // Query if the UITour API is working before we use the API
                     Mozilla.UITour.getConfiguration('sync', function (config) {
 
-                        // Variation #2: Firefox 31+ signed IN to Sync
+                        // Variation #2: Firefox 31+ signed IN to Sync (shows Pocket promo)
                         if (config.setup) {
                             HomePageB.setBodyClass('state-fx-signed-in');
                             HomePageB.state = 'Firefox 31 or Higher: Signed-In';
 
+                            HomePageB.ctaPocket = document.getElementById('pocket-cta');
+                            HomePageB.ctaPocket.addEventListener('click', HomePageB.ctaPocketClick);
```

Comment:
oh, just caught this - should add the default `false` as the last argument here and below?